src/utils/location.ts

```diff
@@ -0,0 +1,3 @@
+export const getLocationOrigin = () => {
```

Comment:
nit: please add a comment explaining why this is in this file. (that we need it for tests, etc)